tests/global-fetch.spec.ts

```diff
@@ -41,27 +42,26 @@ it.afterAll(() => {
 for (const method of ['fetch', 'delete', 'get', 'head', 'patch', 'post', 'put']) {
   it(`${method} should work`, async ({ playwright, server }) => {
     const request = await playwright.request.newContext();
-    const response = await request[method](server.PREFIX + '/simple.json');
+    const response: APIResponse = await request[method](server.PREFIX + '/simple.json');
```

Comment:
why do we need this? isn't tsc smart enough?

Reply:
good catch. I fixed it the other way around by adding `as const` to the for loop values.